### ISSUE TITLE
Invalid root error on Table root

### ISSUE THREAD

**Issue by hirokiosame** [vuejs/eslint-plugin-vue#56]
**Tell us about your environment**

* **ESLint Version:** ^3.19.0
* **eslint-plugin-vue Version:** ^3.2.0
* **Node Version:** v7.9.0

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
	<table>
		<custom-thead></custom-thead>
	</table>
</template>

```

**What did you expect to happen?**
For linting to pass

**What actually happened? Please include the actual, raw output from ESLint.**
`  2:2  error  The template root requires exactly one element  vue/no-invalid-template-root`

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it.

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it.
Vue.js supports it, but it's invalid syntax as HTML. This plugin is using HTML parser (`parse5`), so the `<custom-thead>` moved to outside of the table.

For now, a workaround is that you use correct HTML:

```html
<template>
	<table>
		<thead is="custom-thead"></thead>
	</table>
</template>
```
